[Bug] Failed to build FE on Centos6

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.


### Version

Doris Version: master
OS: Centos 6 x86_64

### What's Wrong?

Failed to build FE on Centos 6.

```shell
[INFO] --- protoc-jar-maven-plugin:3.11.4:run (default) @ fe-core ---
[INFO] Resolving artifact: com.google.protobuf:protoc:3.21.9, platform: linux-x86_64
protoc-jar: executing: [/tmp/protoc939416317298399877.exe, --version]
[INFO] Protoc command: /tmp/protoc939416317298399877.exe
/tmp/protoc939416317298399877.exe: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/protoc939416317298399877.exe)
[INFO] Input directories:
[INFO]     /Programs/doris/fe/../gensrc/proto
[INFO] Output targets:
[INFO]     java: /Programs/doris/fe/fe-core/target/generated-sources (add: main, clean: false, plugin: null, outputOptions: null)
[INFO]     grpc-java: /Programs/doris/fe/fe-core/target/generated-sources (add: main, clean: false, plugin: null, outputOptions: null)
[INFO] Resolving artifact: io.grpc:protoc-gen-grpc-java:1.30.0, platform: linux-x86_64
[INFO]     Processing (java): olap_file.proto
protoc-jar: executing: [/tmp/protoc939416317298399877.exe, -I/Programs/doris/fe/../gensrc/proto, --java_out=/Programs/doris/fe/fe-core/target/generated-sources, /Programs/doris/fe/../gensrc/proto/olap_file.proto]
/tmp/protoc939416317298399877.exe: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/protoc939416317298399877.exe)
[ERROR] /Programs/doris/fe/../gensrc/proto/olap_file.proto [0:0]: /tmp/protoc939416317298399877.exe: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/protoc939416317298399877.exe)
[ERROR] /Programs/doris/fe/../gensrc/proto/olap_file.proto [0:0]:
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for fe-common 1.2-SNAPSHOT:
[INFO]
[INFO] fe-common .......................................... SUCCESS [ 28.794 s]
[INFO] spark-dpp .......................................... SUCCESS [ 38.252 s]
[INFO] fe-core ............................................ FAILURE [01:15 min]
[INFO] hive-udf ........................................... SKIPPED
[INFO] java-udf ........................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:22 min
[INFO] Finished at: 2023-05-04T11:54:33+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.os72:protoc-jar-maven-plugin:3.11.4:run (default) on project fe-core: protoc-jar failed for /Programs/doris/fe/../gensrc/proto/olap_file.proto. Exit code 1 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :fe-core
```

### What You Expected?

Build FE successfully on Centos 6.

### How to Reproduce?

```
bash build.sh --fe --clean
```

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/fe/fe-core/pom.xml
+++ b/fe/fe-core/pom.xml
@@ -29,6 +29,8 @@ under the License.
     <artifactId>fe-core</artifactId>
     <packaging>jar</packaging>
     <properties>
+        <doris.home>${basedir}/../../</doris.home>
+        <doris.thirdparty>${basedir}/../../thirdparty</doris.thirdparty>
         <fe_ut_parallel>1</fe_ut_parallel>
         <antlr4.version>4.9.3</antlr4.version>
         <awssdk.version>2.17.257</awssdk.version>

